nexus/background-task-interface/src/init.rs

```diff
@@ -48,6 +48,7 @@ pub struct BackgroundTasks {
     pub task_alert_dispatcher: Activator,
     pub task_webhook_deliverator: Activator,
     pub task_sp_ereport_ingester: Activator,
+    pub task_chicken_switches_collector: Activator,
```

Comment:
Tiny nit - maybe 

```suggestion
    pub task_chicken_switches_loader: Activator,
```

I'm not sure we've been consistent with this, but I'd expect "_collector" to be a task that goes out and collects things (e.g., inventory), vs "_loader" is a task that reads stuff from the DB.